company/how-to-guides-for-staff/how-to-spend-company-money/README.md

```diff
@@ -68,9 +68,62 @@ For all new staff members, to request a new MacBook, Lenovo or other laptop, ple
 
-#### Mobile Devices: 
+### MOBILE DEVICE POLICY: 
+
+The Mattermost mobile phone policy enables the use of mobile devices for customer-facing roles or those providing repeating on-call service in a secure fashion.
+
+#### Subscription
+
+If you are using your phone primarily for business related matters as defined by your local tax laws, we will reimburse up to 100 USD. Common positions or roles where mobile reimbursement may be needed include the following:
+
+* Customer-facing roles: 
+    * Account Executives
+    * Customer Success Managers
+    * Sales Leadership
+    * Sales Engineering
+* On-call roles: 
+    * Software Design Engineer
+    * Build Engineer
+    * Security Engineer
+    * Site Reliability Engineer
+    * Support Engineers
+
+For all other positions, we will reimburse up to 50 USD.
+If you are unsure if you qualify for a customer facing role or on-call role, reach out to your manager and follow the Manager Approval process.
+
+The company does not currently provide mobile phone subscriptions in a centralized fashion. Subscriptions should be organized by the employee and expensed monthly. Mattermost recommends the usage of a monthly subscription model where possible.
+
+#### Hardware
+The company standard device is an iPhone SE that should be ordered by the employee and expensed. Currently no other platforms are supported.
+
+Usage of a company phone mandatory for the following staff groups:
+* Security Engineer
+
+Usage of a company phone voluntary for the following staff groups:
+* People who are participating in on-call rotations, with approval from their direct manager and not part of a mandatory staff group
+
+Continued usage of personal phones for business is endorsed and acceptable within the voluntary group. Remaining staff groups are currently not eligible for a company mobile phone. The scope of the staff groups may be modified in the future, resulting in changes in mandatory and voluntary groups.
+
+#### Mobile Device Management (MDM)
+
+In order to protect company information, usage of a mobile device management (MDM) solution is mandatory for a company provided phone. An MDM must not be configured when using a personal phone.
+
+When the device has been lost, the MDM solution enables the company to remotely wipe the phone to ensure protection of customer data and additionally enforce security policies on the device. 
+
+#### Process
+
+Each employee can order the hardware and subscription if they are part of an eligible group. Afterwards expense it according to the current process each month for the subscription and once for the hardware.
+
+After the hardware was received, it is mandatory to register with our Mobile Device Management (MDM) provider following this process [link here] and our [Asset Registration Form](https://docs.google.com/forms/d/e/1FAIpQLSfesCJ-XMvXjBxeAYJxvzzbvZHnSv2rH9I48IioJrURNhd7Eg/viewform). This must be completed within 5 business days after the device was received.
```

Comment:
Should we add a link to the process mentioned here

> it is mandatory to register with our Mobile Device Management (MDM) provider following this process [link here]

Reply:
Yes, renamed this to `link pending` since this is still WIP - That is acceptable for the time being, since people get the devices earliest in a few days. I will add this in a separate PR.